development.ini.example

```diff
@@ -327,11 +337,14 @@ reboot_pkgs = kernel kernel-smp kernel-xen-hypervisor kernel-PAE kernel-xen0 ker
 ## https://fedoraproject.org/wiki/Critical_path_package
 ##
 
-# Enable this to query the Fedora Package Database for the list of Critical
-# Path Packages.  If disabled, it'll just use the hardcoded list below.
+# Enable this to query the Fedora Package Database or the Product Definition
+# Center for the list of Critical Path Packages.  If disabled, it'll just use
+# the hardcoded list below.
 #critpath.type = pkgdb
+#critpath.type = pdc
```

Comment:
Let's only define this once and leave it at the default. It would be good to instead mention in the comment that "pdc" means to query product definition center.